.github/ci-scripts/bash/functions-for-build

```diff
@@ -955,3 +973,22 @@ zip_directory() {
   ARTIFACT_FILE="${2}"
   zip -r ${ARTIFACT_FILE} ${DIR}
 }
+
+zip_file() {
```

Comment:
Can we not use `zip_directory` here?

Reply:
Files present in folder are needed at root of zip, `zip_directory` zips the folder itself